### PR TITLE
Impl manage blocks

### PR DESCRIPTION
**Pull request by mmmommm**
ref #108

### CODE REVIEW

File: example_test.go [hunk 1]
```diff
@@ -342,6 +342,35 @@ func ExampleClient_UndoFollowing() {
 	fmt.Println(uf)
 }
 
+func ExampleClient_Blocking() {
+	client := gotwtr.New("key")
+	b, err := client.Blocking(context.Background(), "user_id")
```
Comment (mmmommm) [inline example_test.go:347]: The way this parameter is passed is different from the other examples, but I did it this way because it's hard to tell which id the id refers to.

File: follow_test.go [hunk 2]
```diff
@@ -1537,7 +1537,7 @@ func Test_postFollowing(t *testing.T) {
 					body := `{
 						"errors": [
 							{
-								"message":"Sorry, that page does not exist, code:34"
```
Comment (mmmommm) [inline follow_test.go:1540]: This change was made because golangci-lint was crashing with goconst errors, but I don't understand the behavior and would like to remove it from the linter if possible.

File: example_test.go (hunk 1 above)
Comment (sivchari) [reply example_test.go:347]: Ah, what does it mean ?
I haven't catch your thought, yet.
Please describe more detail.

File: follow_test.go [hunk 3]
```diff
@@ -871,7 +871,7 @@ func Test_following(t *testing.T) {
 						UserName: "TwitterData",
 					},
 				},
-				Meta: gotwtr.FollowsMeta{
```
Comment (sivchari) [inline follow_test.go:874]: Why did you change ?

File: user_option.go [hunk 4]
```diff
@@ -36,22 +36,52 @@ type FollowOption struct {
 	UserFields      []UserField
 }
 
-func (t FollowOption) addQuery(req *http.Request) {
+func (f *FollowOption) addQuery(req *http.Request) {
```
Comment (sivchari) [inline user_option.go:39]: good catch. thanks.

File: follow_test.go (hunk 3 above)
Comment (mmmommm) [reply follow_test.go:874]: In other test codes, meta struct was passed by address, but only here it was passed by value, so I changed it.

File: example_test.go (hunk 1 above)
Comment (mmmommm) [reply example_test.go:347]: Other example code client method func (e.g client.Blocking) passed `id` parameter, but only `id` can't understand what refers to. So i think `user_id` , `target_user_id` , `source_user_id` and etc is better for users.`

Comment (sivchari) [reply example_test.go:347]: Ah, I agree with you. Go ahead 😄
